src/app/components/common/Annotation/Announcement.tsx

```diff
@@ -37,12 +37,11 @@ const CloseButton = styled.div`
 `;
 
 const Announcement = () => {
-  const network = useAppSelector(getStoredNetwork);
-  const [showAnnotation, setShowAnnotation] = useState(true);
+  const [showAnnotation, setShowAnnotation] = useState(false);
 
   const closeAnnotation = () => setShowAnnotation(false);
 
-  if (showAnnotation && network.networkId === GOERLI_NETWORK_ID) {
+  if (showAnnotation) {
     return (
       <div>
         <AnnouncementWrapper><AttentionIcon/>Goerli network support will be deprecated on April 18th, 2024. As of this
```

Comment:
Don't we want to remove the entire thing?

Reply:
done